packages/core/admin/ee/server/controllers/user.js

```diff
@@ -98,35 +95,4 @@ module.exports = {
       data: getService('user').sanitizeUser(updatedUser),
     };
   },
-
-  async deleteOne(ctx) {
```

Comment:
Why did we have the delete-one and delete-many methods in the EE controller? What behavior was extended?

Reply:
Taken from the admin seats PR. I didn't change it but while reviewing I didn't see any use for it